package_manager/contracts/PR/zeppelin/ZeppelinRegistrar.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity 0.4.24;
+
+
+import '../Registrar.sol';
+import 'openzeppelin-zos/contracts/ownership/Ownable.sol';
+
+contract ZeppelinRegistrar is Ownable, Registrar {
+
+  function initialize(ERC137Registry _registry, bytes32 _rootNode) public {
+    Ownable.initialize(msg.sender);
+    Registrar.initialize(_registry, _rootNode);
+  }
+
+  function createName(bytes32 _label, address _owner) public onlyOwner returns (bytes32) {
```

Comment:
I just noted I don't properly understand how access control to the registrar should be. Who should be the owner of the contract? How are we handling _who_ can push new packages?

Reply:
I was proving that we can reuse exactly the same implementation used by Aragon with another permission control flow. In this scenario, the `owner` will be the only one allowed which doesn't make much sense. IMO, we can start with something public which means so anyone can register a package for ZeppelinOS